### ISSUE TITLE
Duplicate keys aren't cool!

### ISSUE THREAD

**Issue by thejh** [zaach/jsonlint#13]
```
$ echo '{"a":1,"a":2}' | jsonlint
The "sys" module is now called "util". It should have a similar interface.
{
  "a": 2
}
$ 
```

**Comment by odino** [comment]
+1

**Comment by pixelastic** [comment]
I would also like to have jsonlint emit a warning on duplicate keys.

**Comment by itsikavidan** [comment]
+1

**Comment by amire80** [comment]
It actually happened to me at least once that a duplicate key written manually by mistake caused a real bug, so it would be nice to get a warning about that. In my case I'd prefer that to be a fatal error in the build that would force me to fix it before deployment.

**Comment by hotoo** [comment]
:+1:

**Comment by mjezzi** [comment]
100% agree. There is no reason to purposefully hard code duplicate keys in a JSON file. A linter should definitely check for this. Via an option is reasonable, considering it's not part of the JSON spec. Any chance this will get added? Need help creating a PR?

**Comment by azamat-sharapov** [comment]
> A pure JavaScript version of the service provided at jsonlint.com.

Judging from readme, this projects claims to work like jsonlint.com, but compared to jsonlint.com - it doesn't detect duplicates.

**Comment by adbl** [comment]
We use this library for linting manually written json files, and this problem caused confusion and time wasted more than once. I'd be happy to have it as an option if it shouldn't be the default behavior...

**Comment by hypery2k** [comment]
any chance to get this feature?

**Comment by Christian-Yang** [comment]
Finally, is there a check that supports duplicate keys?

**Comment by adjohnson916** [comment]
Would love this.

**Comment by WindRider7** [comment]
Would love to see this also.

**Comment by kkor** [comment]
7 years later, still nothing?

**Comment by GGurbanov** [comment]
9 years later, humanity is on the verge of extinction (COVID-19) - still nothing...(

**Comment by FuDesign2008** [comment]
OMG, this issue is still open.

**Comment by jpalardy** [comment]
I stumbled on this thread, for this exact problem.

The [jsonlint-mod](https://github.com/circlecell/jsonlint-mod) fork seems to solve that problem and exposes the same bin.

Posting here in case it helps someone else.

**Comment by rdmarsh** [comment]
I know duplicate keys are valid, but I'd imagine most occurrences are a mistake and unexpected and unwanted.

**Comment by HolgerJeromin** [comment]
https://github.com/prantlf/jsonlint seems to be maintained
